ftwbook/graphicblock/contents/graphicblock.py

```diff
@@ -29,7 +29,7 @@ class IGraphicBlockSchema(Schema):
         title=_(u'label_file', default=u'PDF file'),
         required=True)
 
-    mode(preview='hidden')
+    omitted('preview')
```

Comment:
Awkward question no 1) I wonder what the difference is between omitting the field and removing it totally from the schema?

Reply:
I think we need that field so the migration works as expected.